templates/_base/master/files/container-registries.yaml

```diff
@@ -0,0 +1,30 @@
+filesystem: "root"
```

Comment:
@rphillips 
* why is this file required?
* who uses this ?
* how will we configure it if we somebody needs to configure it for their internal registry ??

/cc @crawford

Reply:
The file is needed since crio gets installed with a default config which only includes the `registry.access.redhat.com` registry. docker.io needs to be added to the registry list to pull in defaulted docker images.

/cc @sjenning

Reply:
updated